barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/generated/AvmMini_helper.cpp

```diff
@@ -0,0 +1,52 @@
+#include "barretenberg/ecc/curves/bn254/fr.hpp"
```

Comment:
Could you move all the non-generated files out of the generated folder?

Reply:
I second this, it will make reviewing work related to pil much easier going forward